lib/ruby_lsp/server.rb

```diff
@@ -207,15 +207,15 @@ module RubyLsp
       Handler::VOID
     end
 
-    on("textDocument/diagnostic") do |request|
+    on("textDocument/diagnostic", parallel: true) do |request|
       uri = request.dig(:params, :textDocument, :uri)
       response = store.cache_fetch(uri, :diagnostics) do |document|
         Requests::Diagnostics.new(uri, document).run
       end
 
       { kind: "full", items: response.map(&:to_lsp_diagnostic) } if response
     end.on_error do |error|
-      if error.is_a?(RuboCop::ValidationError)
+      if error.is_a?(Requests::Support::RuboCopRunner::ConfigurationError)
```

Comment:
This won't work as intended since if `RuboCop` is not available then `RuboCopRunner` won't be available either.